recipes/llnl-units/all/conanfile.py

```diff
@@ -0,0 +1,106 @@
+import os
+
+from conan import ConanFile
+from conan.tools.cmake import CMake, CMakeToolchain
+from conan.tools.files import copy, rm, rmdir, get
+
+
+class UnitsConan(ConanFile):
+    name = "llnl-units"
+    license = "BSD-3"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://units.readthedocs.io"
+    description = (
+        "A run-time C++ library for working with units "
+        "of measurement and conversions between them "
+        "and with string representations of units "
+        "and measurements"
+    )
+    topics = (
+        "units",
+        "dimensions",
+        "quantities",
+        "physical-units",
+        "dimensional-analysis",
+        "run-time",
+    )
+    settings = "os", "compiler", "build_type", "arch"
+    package_type = "library"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "base_type": ["uint32_t", "uint64_t"],
+        "namespace": [None, "ANY"],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "base_type": "uint32_t",
+        "namespace": None,
+    }
+
+    @property
+    def _min_cppstd(self):
+        return 17
```

Comment:
The project refers as C++14 here: https://github.com/LLNL/units/blob/v0.9.1/CMakeLists.txt#L35

Am I missing something?

Reply:
Thanks for the review!

This allows us to use nested namespaces something like `llnl::units`.

It's a nice to have when there are so many "units" libraries out there :)

Reply:
I guess you are referring about that review, right? https://github.com/conan-io/conan-center-index/pull/24459/files/61b135395d3f8be52aeb4b2de35e607113bdacd5#diff-170923aa2ee0d9ef9d7b6df1188a6b29b070ce5e731cb984e06a90fe94de9c6e

I see, but in ConanCenterIndex package as closer are possible to the upstream. In case you see a possible name conflict, I would suggest asking to the upstream changing it first. Thank you explaining your situation 😄